### PR TITLE
Added August 24 check-in

### PR DESCRIPTION
**Pull request by dasanchez**
Venue: Pauper's Pub (subject to change)

### CODE REVIEW

**Review by benhylau — APPROVED** [review]

**Comment by benhylau** [comment]
Travis is really bad at checking links. The link works.

File: _posts/2017-08-24-monthly-checkin.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+locationLink: http://www.openstreetmap.org/node/303251580
+date: 2017-08-24
+startTime: '19:00'
+endTime: '21:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly check-in with Toronto Mesh working groups. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}).  
+Join the {% include icon-matrix.html %} chat for further updates!
+
+A recap of items discussed will be available in our [{{ page.date | date: '%B %d, %Y' }} check-in notes](https://github.com/tomeshnet/documents/blob/master/meeting_notes/{{ page.date | date: "%Y%m%d" }}_check-in.md).
```
Comment (dcwalk) [inline _posts/2017-08-24-monthly-checkin.md]: Do you want to have this here? (only needed for notes) I thought we had a conditional to only show this block after events... once meeting notes were up

**Comment by dcwalk** [comment]
@benhylau -- the meeting notes don't exist yet, so that link is failing legitimately.

**Comment by dcwalk** [comment]
Also -- fine with the location, but pauper's pub doesn't have a patio (or does it?). 

If not, is there somewhere around there with a patio we could do instead?
e.g., Pour Boy or Northwood?

File: _posts/2017-08-24-monthly-checkin.md (hunk 1 above)
Comment (dasanchez) [reply _posts/2017-08-24-monthly-checkin.md]: I'll take off the bit about the notes...I couldn't find a sample entry with the conditional.

**Comment by dasanchez** [comment]
I'll switch it to Northwood.

**Review by dcwalk — APPROVED** [review]

**Comment by dcwalk** [comment]
Looks good, I'll merge in because of the prior approval too :)

**Pull request merged**